src/NUnitEngine/nunit-agent/Program.cs

```diff
@@ -76,6 +75,7 @@ public static int Main(string[] args)
             }
 
             InternalTrace.Initialize($"nunit-agent_{pid}.log", traceLevel);
+            log = InternalTrace.GetLogger(typeof(NUnitTestAgent));
```

Comment:
@mikkelbu - These are the relvant two lines. Previously, line 78 was a static class var - so ran as soon as the class was created. When the log was initialised on line 77, I think the state changed, meaning all later messages using this log variable were lost. Running this after initialisation instead of before means we don't loose the log messages. 😄